Crashlytics/UnitTests/Mocks/FIRCLSMockSymbolResolver.m

```diff
@@ -35,12 +35,12 @@ - (instancetype)init {
   return self;
 }
 
-- (void)addMockFrame:(FIRCLSStackFrame *)frame atAddress:(uint64_t)address {
+- (void)addMockFrame:(FIRStackFrame *)frame atAddress:(uint64_t)address {
   [_frames setObject:frame forKey:@(address)];
 }
 
-- (BOOL)updateStackFrame:(FIRCLSStackFrame *)frame {
-  FIRCLSStackFrame *matchedFrame = [_frames objectForKey:@([frame address])];
+- (BOOL)updateStackFrame:(FIRStackFrame *)frame {
+  FIRStackFrame *matchedFrame = [_frames objectForKey:@([frame address])];
```

Comment:
Nit:  Use the property syntax (frame.address) to make it the code a little more readable.